packages/e2e/features/ui/components/authenticator/form-fields.feature

```diff
@@ -11,38 +11,38 @@ Feature: Form Fields
 
```

Comment:
Cleaned up this file while I was here (we really need to find a prettier config for `.feature` files!).

Reply:
yea, how about this one? https://github.com/cucumber/cucumber-js-pretty-formatter#readme